hypha/templates/includes/login_button.html

```diff
@@ -2,23 +2,68 @@
 
 {% if request.user.is_authenticated %}
     {% if request.path == "/" and request.user.can_access_dashboard %}
-        <a
-            class="button button--transparent button--narrow {{ class }}"
-            href="{{ APPLY_SITE.root_url }}{% url 'dashboard:dashboard' %}"
-        >
-            {% heroicon_micro "user" class="inline align-text-bottom w-4 h-4 me-1" aria_hidden=true %}
-
-            {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
-        </a>
+        <div x-data="{ show: false }" class="relative flex">
+            <button x-on:click="show = ! show" class="button--narrow font-bold text-center flex items-center justify-center px-2 hover:bg-black hover:text-white active:bg-black active:text-white focus:bg-black focus:text-white" type="button">
+                {% heroicon_micro "user-circle" class="inline align-text-bottom w-8 h-8 me-1" aria_hidden=true %}
+                {% blocktrans %}My {{ ORG_SHORT_NAME }}{% endblocktrans %}
+            </button>
+            <div x-show="show" x-transition @click.outside="show = false" class="min-w-36 absolute block bg-white shadow-xl z-20 border-y rounded-sm end-0 top-10">
```

Comment:
I think I prefer without "end-0" so the meny align with the left of the button. Personal and minor.

Reply:
Does not work with short names I found out.